ShotHandler/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/configuration/SpringConfiguration.java

```diff
@@ -0,0 +1,35 @@
+package pl.jakubowskiprzemyslaw.tajgertim.configuration;
+
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import pl.jakubowskiprzemyslaw.tajgertim.services.LoggerService;
+import pl.jakubowskiprzemyslaw.tajgertim.services.QueueService;
+import pl.jakubowskiprzemyslaw.tajgertim.services.SessionService;
+
+@Configuration
+public class SpringConfiguration {
```

Comment:
We would like to wait with this change until #57 will be finished.
It is possible that after packages will be changed to more responsibility oriented, it will be no more public API needed here.
Public API documentation is tracked under #64